sources/common/src/Utils.cpp

```diff
@@ -48,31 +45,11 @@ std::vector<std::string> UTILS::Split(const std::string &rInString,
         {
             outStrings.push_back(outField);
         }
+        else
+        {
+            outStrings.push_back("");
```

Comment:
Nooo why?

Reply:
If you really want to do it like this, just push `outField` unconditionally.

Reply:
Well, it does what it needs to do currently. The function is only used to parse the merger's command line sources. It shall return for all sources the exact and the same number of parameters.

E.g. a,b,c,d,e   - 5 parameters: a, b, c, d, e
       a,b,,,e       - 5 parameters: a,b,"","",e

If we would skip returning the empty string only 3 parameters would be counted. Unfortunately we do not have an integration test for it and I caught it by accident. I will add such a test to a backlog at least for now.